### ISSUE TITLE
[Request An Update]: Add MPXV models to VADR 1.4.2 docker image

### ISSUE THREAD

**Issue by kapsakcj** [StaPH-B/docker-builds#413]
### What container needs an update?

 info here: https://github.com/ncbi/vadr/wiki/Monkeypox-virus-annotation

**Comment by kapsakcj** [comment]
Need to dump model files into `$VADRINSTALLDIR/vadr-models`.

Here's what it currently looks like in the 1.4.2 docker image:
```
$ ls $VADRINSTALLDIR/vadr-models
00NOTES.txt                    NC_008311.vadr.protein.fa.ssi  NC_030793.vadr.protein.fa      NC_044853.vadr.protein.fa.pdb
00README.txt                   NC_008580.vadr.protein.fa      NC_030793.vadr.protein.fa.pdb  NC_044853.vadr.protein.fa.phr
NC_000940.vadr.protein.fa      NC_008580.vadr.protein.fa.pdb  NC_030793.vadr.protein.fa.phr  NC_044853.vadr.protein.fa.pin
NC_000940.vadr.protein.fa.pdb  NC_008580.vadr.protein.fa.phr  NC_030793.vadr.protein.fa.pin  NC_044853.vadr.protein.fa.pot
NC_000940.vadr.protein.fa.phr  NC_008580.vadr.protein.fa.pin  NC_030793.vadr.protein.fa.pot  NC_044853.vadr.protein.fa.psq
NC_000940.vadr.protein.fa.pin  NC_008580.vadr.protein.fa.pot  NC_030793.vadr.protein.fa.psq  NC_044853.vadr.protein.fa.ptf
NC_000940.vadr.protein.fa.pot  NC_008580.vadr.protein.fa.psq  NC_030793.vadr.protein.fa.ptf  NC_044853.vadr.protein.fa.pto
NC_000940.vadr.protein.fa.psq  NC_008580.vadr.protein.fa.ptf  NC_030793.vadr.protein.fa.pto  NC_044853.vadr.protein.fa.ssi
NC_000940.vadr.protein.fa.ptf  NC_008580.vadr.protein.fa.pto  NC_030793.vadr.protein.fa.ssi  NC_044854.vadr.protein.fa
NC_000940.vadr.protein.fa.pto  NC_008580.vadr.protein.fa.ssi  NC_033081.vadr.protein.fa      NC_044854.vadr.protein.fa.pdb
NC_000940.vadr.protein.fa.ssi  NC_010624.vadr.protein.fa      NC_033081.vadr.protein.fa.pdb  NC_044854.vadr.protein.fa.phr
NC_001481.vadr.protein.fa      NC_010624.vadr.protein.fa.pdb  NC_033081.vadr.protein.fa.phr  NC_044854.vadr.protein.fa.pin
NC_001481.vadr.protein.fa.pdb  NC_010624.vadr.protein.fa.phr  NC_033081.vadr.protein.fa.pin  NC_044854.vadr.protein.fa.pot
NC_001481.vadr.protein.fa.phr  NC_010624.vadr.protein.fa.pin  NC_033081.vadr.protein.fa.pot  NC_044854.vadr.protein.fa.psq
NC_001481.vadr.protein.fa.pin  NC_010624.vadr.protein.fa.pot  NC_033081.vadr.protein.fa.psq  NC_044854.vadr.protein.fa.ptf
NC_001481.vadr.protein.fa.pot  NC_010624.vadr.protein.fa.psq  NC_033081.vadr.protein.fa.ptf  NC_044854.vadr.protein.fa.pto
NC_001481.vadr.protein.fa.psq  NC_010624.vadr.protein.fa.ptf  NC_033081.vadr.protein.fa.pto  NC_044854.vadr.protein.fa.ssi
NC_001481.vadr.protein.fa.ptf  NC_010624.vadr.protein.fa.pto  NC_033081.vadr.protein.fa.ssi  NC_044855.vadr.protein.fa
NC_001481.vadr.protein.fa.pto  NC_010624.vadr.protein.fa.ssi  NC_033776.vadr.protein.fa      NC_044855.vadr.protein.fa.pdb
NC_001481.vadr.protein.fa.ssi  NC_011050.vadr.protein.fa      NC_033776.vadr.protein.fa.pdb  NC_044855.vadr.protein.fa.phr
NC_001543.vadr.protein.fa      NC_011050.vadr.protein.fa.pdb  NC_033776.vadr.protein.fa.phr  NC_044855.vadr.protein.fa.pin
NC_001543.vadr.protein.fa.pdb  NC_011050.vadr.protein.fa.phr  NC_033776.vadr.protein.fa.pin  NC_044855.vadr.protein.fa.pot
NC_001543.vadr.protein.fa.phr  NC_011050.vadr.protein.fa.pin  NC_033776.vadr.protein.fa.pot  NC_044855.vadr.protein.fa.psq
NC_001543.vadr.protein.fa.pin  NC_011050.vadr.protein.fa.pot  NC_033776.vadr.protein.fa.psq  NC_044855.vadr.protein.fa.ptf
NC_001543.vadr.protein.fa.pot  NC_011050.vadr.protein.fa.psq  NC_033776.vadr.protein.fa.ptf  NC_044855.vadr.protein.fa.pto
NC_001543.vadr.protein.fa.psq  NC_011050.vadr.protein.fa.ptf  NC_033776.vadr.protein.fa.pto  NC_044855.vadr.protein.fa.ssi
NC_001543.vadr.protein.fa.ptf  NC_011050.vadr.protein.fa.pto  NC_033776.vadr.protein.fa.ssi  NC_044856.vadr.protein.fa
NC_001543.vadr.protein.fa.pto  NC_011050.vadr.protein.fa.ssi  NC_034444.vadr.protein.fa      NC_044856.vadr.protein.fa.pdb
NC_001543.vadr.protein.fa.ssi  NC_011704.vadr.protein.fa      NC_034444.vadr.protein.fa.pdb  NC_044856.vadr.protein.fa.phr
NC_001959.vadr.protein.fa      NC_011704.vadr.protein.fa.pdb  NC_034444.vadr.protein.fa.phr  NC_044856.vadr.protein.fa.pin
NC_001959.vadr.protein.fa.pdb  NC_011704.vadr.protein.fa.phr  NC_034444.vadr.protein.fa.pin  NC_044856.vadr.protein.fa.pot
NC_001959.vadr.protein.fa.phr  NC_011704.vadr.protein.fa.pin  NC_034444.vadr.protein.fa.pot  NC_044856.vadr.protein.fa.psq
NC_001959.vadr.protein.fa.pin  NC_011704.vadr.protein.fa.pot  NC_034444.vadr.protein.fa.psq  NC_044856.vadr.protein.fa.ptf
NC_001959.vadr.protein.fa.pot  NC_011704.vadr.protein.fa.psq  NC_034444.vadr.protein.fa.ptf  NC_044856.vadr.protein.fa.pto
NC_001959.vadr.protein.fa.psq  NC_011704.vadr.protein.fa.ptf  NC_034444.vadr.protein.fa.pto  NC_044856.vadr.protein.fa.ssi
NC_001959.vadr.protein.fa.ptf  NC_011704.vadr.protein.fa.pto  NC_034444.vadr.protein.fa.ssi  NC_044932.vadr.protein.fa
NC_001959.vadr.protein.fa.pto  NC_011704.vadr.protein.fa.ssi  NC_035675.vadr.protein.fa      NC_044932.vadr.protein.fa.pdb
NC_001959.vadr.protein.fa.ssi  NC_012699.vadr.protein.fa      NC_035675.vadr.protein.fa.pdb  NC_044932.vadr.protein.fa.phr
NC_002551.vadr.protein.fa      NC_012699.vadr.protein.fa.pdb  NC_035675.vadr.protein.fa.phr  NC_044932.vadr.protein.fa.pin
NC_002551.vadr.protein.fa.pdb  NC_012699.vadr.protein.fa.phr  NC_035675.vadr.protein.fa.pin  NC_044932.vadr.protein.fa.pot
NC_002551.vadr.protein.fa.phr  NC_012699.vadr.protein.fa.pin  NC_035675.vadr.protein.fa.pot  NC_044932.vadr.protein.fa.psq
NC_002551.vadr.protein.fa.pin  NC_012699.vadr.protein.fa.pot  NC_035675.vadr.protein.fa.psq  NC_044932.vadr.protein.fa.ptf
NC_002551.vadr.protein.fa.pot  NC_012699.vadr.protein.fa.psq  NC_035675.vadr.protein.fa.ptf  NC_044932.vadr.protein.fa.pto
NC_002551.vadr.protein.fa.psq  NC_012699.vadr.protein.fa.ptf  NC_035675.vadr.protein.fa.pto  NC_044932.vadr.protein.fa.ssi
NC_002551.vadr.protein.fa.ptf  NC_012699.vadr.protein.fa.pto  NC_035675.vadr.protein.fa.ssi  NC_045512.vadr.protein.fa
NC_002551.vadr.protein.fa.pto  NC_012699.vadr.protein.fa.ssi  NC_039475.vadr.protein.fa      NC_045512.vadr.protein.fa.pdb
NC_002551.vadr.protein.fa.ssi  NC_017936.vadr.protein.fa      NC_039475.vadr.protein.fa.pdb  NC_045512.vadr.protein.fa.phr
NC_002615.vadr.protein.fa      NC_017936.vadr.protein.fa.pdb  NC_039475.vadr.protein.fa.phr  NC_045512.vadr.protein.fa.pin
NC_002615.vadr.protein.fa.pdb  NC_017936.vadr.protein.fa.phr  NC_039475.vadr.protein.fa.pin  NC_045512.vadr.protein.fa.pot
NC_002615.vadr.protein.fa.phr  NC_017936.vadr.protein.fa.pin  NC_039475.vadr.protein.fa.pot  NC_045512.vadr.protein.fa.psq
NC_002615.vadr.protein.fa.pin  NC_017936.vadr.protein.fa.pot  NC_039475.vadr.protein.fa.psq  NC_045512.vadr.protein.fa.ptf
NC_002615.vadr.protein.fa.pot  NC_017936.vadr.protein.fa.psq  NC_039475.vadr.protein.fa.ptf  NC_045512.vadr.protein.fa.pto
NC_002615.vadr.protein.fa.psq  NC_017936.vadr.protein.fa.ptf  NC_039475.vadr.protein.fa.pto  NC_045512.vadr.protein.fa.ssi
NC_002615.vadr.protein.fa.ptf  NC_017936.vadr.protein.fa.pto  NC_039475.vadr.protein.fa.ssi  NC_045762.vadr.protein.fa
NC_002615.vadr.protein.fa.pto  NC_017936.vadr.protein.fa.ssi  NC_039476.vadr.protein.fa      NC_045762.vadr.protein.fa.pdb
NC_002615.vadr.protein.fa.ssi  NC_019712.vadr.protein.fa      NC_039476.vadr.protein.fa.pdb  NC_045762.vadr.protein.fa.phr
NC_004064.vadr.protein.fa      NC_019712.vadr.protein.fa.pdb  NC_039476.vadr.protein.fa.phr  NC_045762.vadr.protein.fa.pin
NC_004064.vadr.protein.fa.pdb  NC_019712.vadr.protein.fa.phr  NC_039476.vadr.protein.fa.pin  NC_045762.vadr.protein.fa.pot
NC_004064.vadr.protein.fa.phr  NC_019712.vadr.protein.fa.pin  NC_039476.vadr.protein.fa.pot  NC_045762.vadr.protein.fa.psq
NC_004064.vadr.protein.fa.pin  NC_019712.vadr.protein.fa.pot  NC_039476.vadr.protein.fa.psq  NC_045762.vadr.protein.fa.ptf
NC_004064.vadr.protein.fa.pot  NC_019712.vadr.protein.fa.psq  NC_039476.vadr.protein.fa.ptf  NC_045762.vadr.protein.fa.pto
NC_004064.vadr.protein.fa.psq  NC_019712.vadr.protein.fa.ptf  NC_039476.vadr.protein.fa.pto  NC_045762.vadr.protein.fa.ssi
NC_004064.vadr.protein.fa.ptf  NC_019712.vadr.protein.fa.pto  NC_039476.vadr.protein.fa.ssi  RELEASE-NOTES.txt
NC_004064.vadr.protein.fa.pto  NC_019712.vadr.protein.fa.ssi  NC_039477.vadr.protein.fa      build-info.txt
NC_004064.vadr.protein.fa.ssi  NC_024031.vadr.protein.fa      NC_039477.vadr.protein.fa.pdb  calici.cm
NC_004541.vadr.protein.fa      NC_024031.vadr.protein.fa.pdb  NC_039477.vadr.protein.fa.phr  calici.cm.i1f
NC_004541.vadr.protein.fa.pdb  NC_024031.vadr.protein.fa.phr  NC_039477.vadr.protein.fa.pin  calici.cm.i1i
NC_004541.vadr.protein.fa.phr  NC_024031.vadr.protein.fa.pin  NC_039477.vadr.protein.fa.pot  calici.cm.i1m
NC_004541.vadr.protein.fa.pin  NC_024031.vadr.protein.fa.pot  NC_039477.vadr.protein.fa.psq  calici.cm.i1p
NC_004541.vadr.protein.fa.pot  NC_024031.vadr.protein.fa.psq  NC_039477.vadr.protein.fa.ptf  calici.fa
NC_004541.vadr.protein.fa.psq  NC_024031.vadr.protein.fa.ptf  NC_039477.vadr.protein.fa.pto  calici.fa.ndb
NC_004541.vadr.protein.fa.ptf  NC_024031.vadr.protein.fa.pto  NC_039477.vadr.protein.fa.ssi  calici.fa.nhr
NC_004541.vadr.protein.fa.pto  NC_024031.vadr.protein.fa.ssi  NC_039897.vadr.protein.fa      calici.fa.nin
NC_004541.vadr.protein.fa.ssi  NC_024078.vadr.protein.fa      NC_039897.vadr.protein.fa.pdb  calici.fa.not
NC_004542.vadr.protein.fa      NC_024078.vadr.protein.fa.pdb  NC_039897.vadr.protein.fa.phr  calici.fa.nsq
NC_004542.vadr.protein.fa.pdb  NC_024078.vadr.protein.fa.phr  NC_039897.vadr.protein.fa.pin  calici.fa.ntf
NC_004542.vadr.protein.fa.phr  NC_024078.vadr.protein.fa.pin  NC_039897.vadr.protein.fa.pot  calici.fa.nto
NC_004542.vadr.protein.fa.pin  NC_024078.vadr.protein.fa.pot  NC_039897.vadr.protein.fa.psq  calici.fa.ssi
NC_004542.vadr.protein.fa.pot  NC_024078.vadr.protein.fa.psq  NC_039897.vadr.protein.fa.ptf  calici.list
NC_004542.vadr.protein.fa.psq  NC_024078.vadr.protein.fa.ptf  NC_039897.vadr.protein.fa.pto  calici.minfo
NC_004542.vadr.protein.fa.ptf  NC_024078.vadr.protein.fa.pto  NC_039897.vadr.protein.fa.ssi  calici.pt.hmm
NC_004542.vadr.protein.fa.pto  NC_024078.vadr.protein.fa.ssi  NC_040674.vadr.protein.fa      calici.pt.hmm.h3f
NC_004542.vadr.protein.fa.ssi  NC_025676.vadr.protein.fa      NC_040674.vadr.protein.fa.pdb  calici.pt.hmm.h3i
NC_006269.vadr.protein.fa      NC_025676.vadr.protein.fa.pdb  NC_040674.vadr.protein.fa.phr  calici.pt.hmm.h3m
NC_006269.vadr.protein.fa.pdb  NC_025676.vadr.protein.fa.phr  NC_040674.vadr.protein.fa.pin  calici.pt.hmm.h3p
NC_006269.vadr.protein.fa.phr  NC_025676.vadr.protein.fa.pin  NC_040674.vadr.protein.fa.pot  noro.cm
NC_006269.vadr.protein.fa.pin  NC_025676.vadr.protein.fa.pot  NC_040674.vadr.protein.fa.psq  noro.cm.i1f
NC_006269.vadr.protein.fa.pot  NC_025676.vadr.protein.fa.psq  NC_040674.vadr.protein.fa.ptf  noro.cm.i1i
NC_006269.vadr.protein.fa.psq  NC_025676.vadr.protein.fa.ptf  NC_040674.vadr.protein.fa.pto  noro.cm.i1m
NC_006269.vadr.protein.fa.ptf  NC_025676.vadr.protein.fa.pto  NC_040674.vadr.protein.fa.ssi  noro.cm.i1p
NC_006269.vadr.protein.fa.pto  NC_025676.vadr.protein.fa.ssi  NC_040876.vadr.protein.fa      noro.fa
NC_006269.vadr.protein.fa.ssi  NC_027122.vadr.protein.fa      NC_040876.vadr.protein.fa.pdb  noro.fa.ndb
NC_006554.vadr.protein.fa      NC_027122.vadr.protein.fa.pdb  NC_040876.vadr.protein.fa.phr  noro.fa.nhr
NC_006554.vadr.protein.fa.pdb  NC_027122.vadr.protein.fa.phr  NC_040876.vadr.protein.fa.pin  noro.fa.nin
NC_006554.vadr.protein.fa.phr  NC_027122.vadr.protein.fa.pin  NC_040876.vadr.protein.fa.pot  noro.fa.not
NC_006554.vadr.protein.fa.pin  NC_027122.vadr.protein.fa.pot  NC_040876.vadr.protein.fa.psq  noro.fa.nsq
NC_006554.vadr.protein.fa.pot  NC_027122.vadr.protein.fa.psq  NC_040876.vadr.protein.fa.ptf  noro.fa.ntf
NC_006554.vadr.protein.fa.psq  NC_027122.vadr.protein.fa.ptf  NC_040876.vadr.protein.fa.pto  noro.fa.nto
NC_006554.vadr.protein.fa.ptf  NC_027122.vadr.protein.fa.pto  NC_040876.vadr.protein.fa.ssi  noro.fa.ssi
NC_006554.vadr.protein.fa.pto  NC_027122.vadr.protein.fa.ssi  NC_044045.vadr.protein.fa      noro.list
NC_006554.vadr.protein.fa.ssi  NC_029645.vadr.protein.fa      NC_044045.vadr.protein.fa.pdb  noro.minfo
NC_006875.vadr.protein.fa      NC_029645.vadr.protein.fa.pdb  NC_044045.vadr.protein.fa.phr  noro.pt.hmm
NC_006875.vadr.protein.fa.pdb  NC_029645.vadr.protein.fa.phr  NC_044045.vadr.protein.fa.pin  noro.pt.hmm.h3f
NC_006875.vadr.protein.fa.phr  NC_029645.vadr.protein.fa.pin  NC_044045.vadr.protein.fa.pot  noro.pt.hmm.h3i
NC_006875.vadr.protein.fa.pin  NC_029645.vadr.protein.fa.pot  NC_044045.vadr.protein.fa.psq  noro.pt.hmm.h3m
NC_006875.vadr.protein.fa.pot  NC_029645.vadr.protein.fa.psq  NC_044045.vadr.protein.fa.ptf  noro.pt.hmm.h3p
NC_006875.vadr.protein.fa.psq  NC_029645.vadr.protein.fa.ptf  NC_044045.vadr.protein.fa.pto  sarscov2.cm
NC_006875.vadr.protein.fa.ptf  NC_029645.vadr.protein.fa.pto  NC_044045.vadr.protein.fa.ssi  sarscov2.cm.i1f
NC_006875.vadr.protein.fa.pto  NC_029645.vadr.protein.fa.ssi  NC_044046.vadr.protein.fa      sarscov2.cm.i1i
NC_006875.vadr.protein.fa.ssi  NC_029646.vadr.protein.fa      NC_044046.vadr.protein.fa.pdb  sarscov2.cm.i1m
NC_007916.vadr.protein.fa      NC_029646.vadr.protein.fa.pdb  NC_044046.vadr.protein.fa.phr  sarscov2.cm.i1p
NC_007916.vadr.protein.fa.pdb  NC_029646.vadr.protein.fa.phr  NC_044046.vadr.protein.fa.pin  sarscov2.fa
NC_007916.vadr.protein.fa.phr  NC_029646.vadr.protein.fa.pin  NC_044046.vadr.protein.fa.pot  sarscov2.fa.ndb
NC_007916.vadr.protein.fa.pin  NC_029646.vadr.protein.fa.pot  NC_044046.vadr.protein.fa.psq  sarscov2.fa.nhr
NC_007916.vadr.protein.fa.pot  NC_029646.vadr.protein.fa.psq  NC_044046.vadr.protein.fa.ptf  sarscov2.fa.nin
NC_007916.vadr.protein.fa.psq  NC_029646.vadr.protein.fa.ptf  NC_044046.vadr.protein.fa.pto  sarscov2.fa.not
NC_007916.vadr.protein.fa.ptf  NC_029646.vadr.protein.fa.pto  NC_044046.vadr.protein.fa.ssi  sarscov2.fa.nsq
NC_007916.vadr.protein.fa.pto  NC_029646.vadr.protein.fa.ssi  NC_044047.vadr.protein.fa      sarscov2.fa.ntf
NC_007916.vadr.protein.fa.ssi  NC_029647.vadr.protein.fa      NC_044047.vadr.protein.fa.pdb  sarscov2.fa.nto
NC_008311.vadr.protein.fa      NC_029647.vadr.protein.fa.pdb  NC_044047.vadr.protein.fa.phr  sarscov2.fa.ssi
NC_008311.vadr.protein.fa.pdb  NC_029647.vadr.protein.fa.phr  NC_044047.vadr.protein.fa.pin  sarscov2.hmm
NC_008311.vadr.protein.fa.phr  NC_029647.vadr.protein.fa.pin  NC_044047.vadr.protein.fa.pot  sarscov2.hmm.h3f
NC_008311.vadr.protein.fa.pin  NC_029647.vadr.protein.fa.pot  NC_044047.vadr.protein.fa.psq  sarscov2.hmm.h3i
NC_008311.vadr.protein.fa.pot  NC_029647.vadr.protein.fa.psq  NC_044047.vadr.protein.fa.ptf  sarscov2.hmm.h3m
NC_008311.vadr.protein.fa.psq  NC_029647.vadr.protein.fa.ptf  NC_044047.vadr.protein.fa.pto  sarscov2.hmm.h3p
NC_008311.vadr.protein.fa.ptf  NC_029647.vadr.protein.fa.pto  NC_044047.vadr.protein.fa.ssi  sarscov2.minfo
NC_008311.vadr.protein.fa.pto  NC_029647.vadr.protein.fa.ssi  NC_044853.vadr.protein.fa
```

**Comment by kapsakcj** [comment]
working on this now, will publish a new branch very soon

**Comment by kapsakcj** [comment]
opened PR #414 to address this

**Comment by kapsakcj** [comment]
closed via merging #414
